### DISCUSSION TITLE
More scratchpads

### DISCUSSION THREAD

**Discussion by rebdeg** [hyprwm/Hyprland#657] [Q&A] [answered]
Today I have been playing around with a scratchpad on my system, works a described. However:
1. Is there an option to have more than one scratchpad
2. Can scratchpads be defined by rules, size, floating etc.

**Comment by vaxerski** [comment, marked as answer]
1. no
2. wdym? you can control its size with layout options, see your layout's wiki page

**Comment by wuyoli** [comment]
Why not multiple scratchpads?
Because noone implemented it yet or because you don't want it? (In the latter case: why?)

**Comment by ingoau** [comment]
Looks like special workspaces can do this
```
bind = $mainMod, A, togglespecialworkspace, special
bind = $mainMod SHIFT, A, movetoworkspace, special

bind = $mainMod, Return, togglespecialworkspace, terminal
```
Not sure if this was implemented when this question was posted but posting here for anyone coming across this thread.
